apiary.apib

```diff
@@ -8187,7 +8187,7 @@ Gets stats for a given Run Id. Provides the count of imported/exported tables an
 ###Create [POST /v2/storage/triggers/]
 Created a trigger listening on set of tables
 
-Token has to be [*master*](https://help.keboola.com/storage/tokens/) (then `runWithTokenId` can be also master). If it isn't master, it has to have `canManageBuckets` permission or `componentAccess` permission for component `keboola.orchestrator`. Applies also on Update and Delete operation
+Token has to be [*master*](https://help.keboola.com/storage/tokens/) (then `runWithTokenId` can be also master). If it isn't master, it has to have `canManageBuckets` permission or `componentAccess` permission for the component which you want to trigger. This also applies when updating and deleting triggers.
```

Comment:
```suggestion
Token has to be [*master*](https://help.keboola.com/storage/tokens/) (then `runWithTokenId` can be also master). If it isn't master, it has to have `canManageBuckets` permission or `componentAccess` permission for the component which you want to trigger. This also applies when updating and deleting triggers. When updating the component of trigger you need permissions for both the current component and the component you're updating the trigger to. 
```

Reply:
Jo to je pravda ze to by to bolo matuce. 👍

Reply:
Pridal som